cvat-sdk/cvat_sdk/core/proxies/tasks.py

```diff
@@ -219,31 +219,24 @@ def download_chunk(
     def download_frames(
         self,
         frame_ids: Sequence[int],
+        image_extension: str,
         *,
-        outdir: StrPath = ".",
+        outdir: str = ".",
         quality: str = "original",
         filename_pattern: str = "frame_{frame_id:06d}{frame_ext}",
     ) -> Optional[List[Image.Image]]:
         """
-        Download the requested frame numbers for a task and save images as outdir/filename_pattern
+        Download the requested frame numbers for a job and save images as outdir/filename_pattern
```

Comment:
Why this has been changed? There should be another similar function in `proxies/job.py`.

Reply:
I will fix it